addons/mail/models/mail_message.py

```diff
@@ -969,6 +969,7 @@ def message_format(self, format_reply=True):
                 'is_discussion': message_sudo.subtype_id.id == com_id,
                 'subtype_description': message_sudo.subtype_id.description,
                 'is_notification': vals['message_type'] == 'user_notification',
+                'partners_data': list(map(lambda p: {'id': p.id, 'name': p.name}, message_sudo.partner_ids)),
```

Comment:
Not a mistake but could use iterator rather than list and map: `[{...} for p in ...]`